src/pages/go/quickstart.mdx

```diff
@@ -79,36 +70,28 @@ If you want the generated metrics to contain the function success rate, you must
 
 Install the go generator using `go install` as usual:
 
-```bash
+```bash filename="Shell"
 go install https://github.com/autometrics-dev/autometrics-go/cmd/autometrics
```

Comment:
The `@latest` part forces to install the latest tagged version, so it tends to be more stable

```suggestion
go install https://github.com/autometrics-dev/autometrics-go/cmd/autometrics@latest
```